app/helpers/statistics_helper.rb

```diff
@@ -4,13 +4,21 @@
 
 module StatisticsHelper
   def registrations_status statistics
-    if statistics.open_registrations?
+    if !!statistics.open_registrations?
```

Comment:
Yeah, problem is : function recieve a symbol, so, as is is not a fasly value, it is always considered true, even if registrations are closed. `!!` "operator" forces cast to boolean. Works well ;)